assets/src/infrastructure/ui/inputs/dateTime/Timepicker/style.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../style.scss';
+
+.ee-time-picker {
+	[data-baseweb='select'] {
+		align-items: center;
+		display: flex;
+		min-height: 1.875rem;
+
+		> div {
+			background-color: transparent;
+			padding: 0 0.5rem;
+
+			@extend .base-inputs-styles;
+			&:hover {
+				@extend .input-select-textarea-hover;
+			}
+			&:focus {
+				@extend .input-select-textarea-focus;
+			}
+			> div {
+				padding: 0;
+				> div {
+					display: contents;
```

Comment:
items are not aligned w/o this prop:

![image](https://user-images.githubusercontent.com/1477453/80974477-72db4f00-8e29-11ea-8511-85a8c6a78b12.png)

Reply:
didn't knew about this property of display until today
***
```
display: contents;—a magical new display value that essentially makes the container disappear, making the child elements children of the element the next level up in the DOM.
```
https://css-tricks.com/get-ready-for-display-contents/

Reply:
ooo i didn't know about that either... COOL